AdvLoggerPkg/Library/AdvancedLoggerLib/MmCoreArm/AdvancedLoggerLib.c

```diff
@@ -87,8 +39,8 @@ ValidateInfoBlock (
   PcdAdvancedLoggerCarBase     -- NOT USED, leave at default

```

Comment:
I noticed "PcdAdvancedLoggerPages" is stated as being assumed to greater than 64KB. Is there a hard constraint on that somewhere or is it what seems reasonable for a log buffer?

Reply:
Yeah, this comments seems to have originated in the ARM BaseLib, but i don't see any enforcement or reason given. So unsure why this is here.